src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -355,52 +442,48 @@ private void applyStrictStrandFilter(final M2FiltersArgumentCollection MTFAC, fi
             return;
         }
 
-        final Genotype tumorGenotype = vc.getGenotype(tumorSample);
-        if (! tumorGenotype.hasExtendedAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY)) {
-            return;
-        }
-        final int[] strandBiasCounts = GATKProtectedVariantContextUtils.getAttributeAsIntArray(tumorGenotype, GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY, ()->null, -1);
+        final MutableInt altForwardCount = new MutableInt(0);
+        final MutableInt altReverseCount = new MutableInt(0);
```

Comment:
What's the benefit of changing this to a MutableInt?

Reply:
I didn't want the array anymore because with a pass over samples requires it seemed too opaque to use an array to encapsulate two numbers.  And once they're two numbers I like the MutableInt because it's final and it makes it more explicit that the for loop below is intended to grow the counts sample-by-sample.  But, I mean, at the end of the day this is my own idiosyncratic preference and I can't really justify it beyond all argument.  Do you think I should keep it an int[]?

Reply:
The way we have it in other places in the code is as an int array with constants describing the indices in the array as totalReadCounts[ALT_FORWARD_COUNT] or similar.  I personally prefer primitives from a memory scalability perspective, but given that this is just two values readability is more important. If you think it's more clear that this is a running total if it's a final MutableInt, then I can be on board.

Reply:
I'm a dunce at reading anything that resembles C.  It's clearer to me this way.